llrt_core/src/modules/http/fetch.rs

```diff
@@ -131,6 +134,50 @@ pub(crate) fn init(ctx: &Ctx<'_>, globals: &Object) -> Result<()> {
     Ok(())
 }
 
+fn parse_data_url<'js>(ctx: &Ctx<'js>, data_url: &String) -> Result<Response<'js>> {
+    let (mime_type, data) = data_url
+        .trim_start_matches("data:")
```

Comment:
Here we just want to remove the `data:` that is not needed for processing. As far as I know, the code would be a bit verbose to remove it in any way other than `trim_start_matches`.

Reply:
we should pass a string slice to the function instead of a string reference:
```rust
fn parse_data_url<'js>(ctx: &Ctx<'js>, data_url: &str){
    let (mime_type, data) = data_url["data:".len()..].split_once(',')...
}
```

Reply:
Thank you very much. I have fixed it with this idea.